packages/web/src/components/cells/title.tsx

```diff
@@ -15,8 +15,11 @@ export default function TitleCell(props: {
   }
 
   return (
-    <div id={`cell-${cell.id}`} className="mb-4">
-      <EditableH1 text={cell.text} className="title" onUpdated={updateCell} />
-    </div>
+    <EditableH1
+      text={cell.text}
+      id={`cell-title-${cell.id}`}
```

Comment:
`cell-${cell.id}` is used to target this element for scrollTo purposes, so I believe this change will break that behavior?

Reply:
prior to this change there was multiple cell-${cell.id} instances which was doing weird things

Reply:
the ids are unique so this shouldn't be problematic. What were the weird things that were happening?